deepchem/metalearning/tests/test_maml_pytorch.py

```diff
@@ -0,0 +1,119 @@
+import deepchem as dc
+import numpy as np
+import pytest
+from flaky import flaky
+import unittest
+from deepchem.metalearning.torch_maml import MetaLearner, MAML
+
+try:
+    import torch
+    import torch.nn.functional as F
+
+    has_pytorch = True
+except:
+    has_pytorch = False
+
+
+class TestMAML(unittest.TestCase):
+
+    @flaky
+    @pytest.mark.torch
+    def test_maml_pytorch(self):
+
+        class SineLearner(MetaLearner):
```

Comment:
Maybe move this out of the test and into the try/catch block above

Reply:
Jax tests are passing now sir.